### ISSUE TITLE
Garage Door Track Rendering

### ISSUE THREAD

**Issue by TedCC1209** [CCDevNelson/Snap-Updates#286]
Can we add a garage door track rendering/object that we can add to scenes to work around? @SeanRadvan is this something you can create? 

Thanks, 
Ted

**Comment by SeanRadvan** [comment]
If its simply creating the rendering for a rail, most likely I should be able to handle that. If you want it be able to interact with cabinets for things like fill to wall, that might take a bit more work. because where the tracks sit in a garage can vary wildly, we would want some sort of dialogue box to adjust its height / length. Not terribly difficult but I would probably need to teddy / J to help me.

**Comment by TedCC1209** [comment]
Yes let's start with just a rendering of the rail/track with the option to adjust the height and length of the track as well as distance off the wall because we want this to be "attached"/"connected" with the wall that the cabinets are going to go on that are having to work around it. Let me know if that makes sense--we can start with this for now.

Thanks, Ted
